website/source/docs/commands/snapshot/agent.html.markdown.erb

```diff
@@ -256,6 +261,18 @@ The following example IAM policy document assumes that the `aws-s3-bucket` is `c
 }
 ```
 
+#### Azure Blob Storage options
+
+From Consul Enterprise version `1.4.X+` onwards, you can store snapshots in Azure Blob storage.
+
+* `azure-blob-account_name` and `-azure-blob-account_key` - These arguments supply
+  authentication information for connecting to Azure Blob storage.
+
+* `azure-blob-container_name` - Container to use. Required for Azure blob storage, and setting this
+  disables local storage.
+
+** Note: This currently only works on non-Solaris platforms due to library limitations **
```

Comment:
Should this line come before the flags? If it's worth the disclaimer maybe it should be at/near the top.

Reply:
Fixed! 😄

Reply:
Thanks both!